CHANGES.md

```diff
@@ -19,6 +19,11 @@ OpenSSL Releases
  - [OpenSSL 1.0.0](#openssl-100)
  - [OpenSSL 0.9.x](#openssl-09x)
 
+HPKE
+----
+
+ * This is sftcd's fork of OpenSSL to implement Hybrid Public Key Encryption (HPKE).
```

Comment:
This needs to be converted into a proper CHANGES.md entry

Reply:
True. Will address when I get to fix up the documentation stuff related to these changes.